mw-webapp/src/logic/waysTable/OwnedWaysTable.tsx

```diff
@@ -0,0 +1,47 @@
+import {useEffect, useState} from "react";
+import {WayPreviewDAL} from "src/dataAccessLogic/WayPreviewDAL";
+import {columns} from "src/logic/waysTable/columns";
+import {WaysTable} from "src/logic/waysTable/WaysTable";
+import {WayPreview} from "src/model/businessModelPreview/WayPreview";
+
+export type OwnedWaysTableProps = {
```

Comment:
we use namingoewWays in models so need to use the same naming everywhere

Reply:
Done